templates/base/Dockerfile

```diff
@@ -1,17 +1,19 @@
-FROM --platform=${TARGETPLATFORM} ubuntu:jammy AS ubuntu-jammy
+FROM --platform=${TARGETPLATFORM} ubuntu:noble AS ubuntu-noble
 
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes adduser bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
 	apt-get upgrade --yes && \
 	apt-get clean
 
-RUN adduser --disabled-password --home /home/liferay liferay --uid 1000 && \
+RUN userdel ubuntu && \
+	adduser --disabled-password --home /home/liferay liferay --uid 1000 && \
 	addgroup liferay liferay && \
```

Comment:
@Wykenz, I realized that this line was removed from your last pull request, please see https://github.com/natocesarrego/liferay-docker/pull/75/commits/f6c1ca8406cc08791b0ae29ac611085951efd043. Was it an intentional change?

Reply:
@natocesarrego Yes, it gave an error on noble so I removed it, but as I saw the user and group settings works as expected